test/src/test/java/org/corfudb/integration/LogReplicationAbstractIT.java

```diff
@@ -260,6 +270,53 @@ public void testEndToEndSnapshotAndLogEntrySyncUFO(int totalNumMaps, boolean dis
         }
     }
 
+    private void triggerSnapshotAndTestRemainingEntries() throws Exception{
+
+        // enforce a snapshot sync
+        Table<ExampleSchemas.ClusterUuidMsg, ExampleSchemas.ClusterUuidMsg, ExampleSchemas.ClusterUuidMsg> configTable =
+                corfuStoreActive.openTable(
+                        DefaultClusterManager.CONFIG_NAMESPACE, DefaultClusterManager.CONFIG_TABLE_NAME,
+                        ExampleSchemas.ClusterUuidMsg.class, ExampleSchemas.ClusterUuidMsg.class, ExampleSchemas.ClusterUuidMsg.class,
+                        TableOptions.fromProtoSchema(ExampleSchemas.ClusterUuidMsg.class)
+                );
+        try (TxnContext txn = corfuStoreActive.txn(DefaultClusterManager.CONFIG_NAMESPACE)) {
+            txn.putRecord(configTable, DefaultClusterManager.OP_ENFORCE_SNAPSHOT_FULL_SYNC,
+                    DefaultClusterManager.OP_ENFORCE_SNAPSHOT_FULL_SYNC, DefaultClusterManager.OP_ENFORCE_SNAPSHOT_FULL_SYNC);
+            txn.commit();
+        }
+
+        // Sleep, so we have remainingEntries populated from the scheduled polling task instead of
+        // from method that marks snapshot complete
+        TimeUnit.SECONDS.sleep(longInterval);
+
+        corfuStoreActive.openTable(LogReplicationMetadataManager.NAMESPACE,
+                LogReplicationMetadataManager.REPLICATION_STATUS_TABLE,
+                LogReplicationMetadata.ReplicationStatusKey.class,
+                LogReplicationMetadata.ReplicationStatusVal.class,
+                null,
+                TableOptions.fromProtoSchema(LogReplicationMetadata.ReplicationStatusVal.class));
+
+        LogReplicationMetadata.ReplicationStatusKey key =
+                LogReplicationMetadata.ReplicationStatusKey
+                        .newBuilder()
+                        .setClusterId(DefaultClusterConfig.getStandbyClusterId())
+                        .build();
+        LogReplicationMetadata.ReplicationStatusVal replicationStatusVal;
+        while(true) {
+            try (TxnContext txn = corfuStoreActive.txn(LogReplicationMetadataManager.NAMESPACE)) {
+                replicationStatusVal = (LogReplicationMetadata.ReplicationStatusVal) txn.getRecord("LogReplicationStatus", key).getPayload();
+                txn.commit();
+                if (replicationStatusVal.getSnapshotSyncInfo().getType().equals(LogReplicationMetadata.SnapshotSyncInfo.SnapshotSyncType.FORCED) &&
+                        replicationStatusVal.getSnapshotSyncInfo().getStatus().equals(LogReplicationMetadata.SyncStatus.COMPLETED)) {
```

Comment:
We do the first log_entry sync and ensure the data has been replicated to standby. Wouldn't that be enough?
If nothing is sent in the first log entry sync, then the assert that checks the standby would fail.

Reply:
sure, if we're checking something was sent on log_entry_sync this should indirectly be enough!